app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,13 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter android:label="@string/processout_return">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="yourapp"
```

Comment:
How will this be set by the merchant?

Reply:
This is explained [here](https://github.com/processout/documentation/pull/27/files#diff-ab7929d3686557b473a1f1563a86bc07R120)